cdk/resources/DeviceShadow.ts

```diff
@@ -93,6 +95,36 @@ export class DeviceShadow extends Construct {
 		scheduler.addTarget(new EventTargets.LambdaFunction(prepareDeviceShadow))
 		shadowQueue.grantSendMessages(prepareDeviceShadow)
 
+		const policies = nRFCloudAccounts
+			.map((account) => [
+				new IAM.PolicyStatement({
+					actions: ['ssm:GetParameter'],
+					resources: [
+						`arn:aws:ssm:${Stack.of(this).region}:${
+							Stack.of(this).account
+						}:parameter/${Stack.of(this).stackName}/thirdParty/${account}/*`,
```

Comment:
Simplify this by using wildcard.

```suggestion
						}:parameter/${Stack.of(this).stackName}/thirdParty/*`,
```